importing-data-5.html

```diff
@@ -86,58 +121,98 @@
 
 </li>
 
-<li><p class="first">Mouse-over the thumbnails of successfully imported images to see a preview.</p>
+<li><p class="first">Import progress will be shown in the <span class="bold">Import # 1</span> tab.</p>
```

Comment:
**If** you're feeling pedantic there shouldn't be a space: `Import #1` not `Import # 1`